src/h5web/visualizations/shared/hooks.ts

```diff
@@ -18,20 +24,28 @@ export function useBaseArray<T>(
   }, [rawDims, value]);
 }
 
+export function useMappedArray<T>(
+  baseArray: ndarray<T>,
+  mapping: DimensionMapping
+): ndarray<T>;
+export function useMappedArray<T>(
+  baseArray: ndarray<T> | undefined,
+  mapping: DimensionMapping
+): ndarray<T> | undefined;
+
 export function useMappedArray<T>(
   baseArray: ndarray<T> | undefined,
-  mapperState: DimensionMapping
+  mapping: DimensionMapping
 ): ndarray<T> | undefined {
   return useMemo(() => {
-    if (!baseArray || !mapperState) {
+    if (!baseArray || !mapping) {
       return baseArray;
     }
```

Comment:
```js
    if (!baseArray) {
      return undefined;
    }
```
`mapping` cannot be `undefined` anymore, right ?

Reply:
Indeed, because `DimensionMapping` no longer includes `undefined`.